portal-api/api-tests/v1/users/post-sign-in-link.api-test.js

```diff
@@ -130,6 +137,13 @@ describe('POST /users/me/sign-in-link', () => {
       const userInDb = await databaseHelper.getUserById(partiallyLoggedInUserId);
       expect(userInDb.signInLinkSendCount).toBe(4);
     });
+
+    it('deletes any existing sign in token data', async () => {
+      await sendSignInLink();
+
+      const userInDb = await databaseHelper.getUserById(partiallyLoggedInUserId);
+      expect(userInDb.signInToken).toBeUndefined();
```

Comment:
Can we also please add as another conditional check.

```javascript
expect(userInDb.signInLinkSendCount).toBe(3);
```

Reply:
We've got a test that does this above - `increments the signInLinkSendCount`